src/supermarket/app/views/api/v1/cookbook_versions/_cookbook_version.json.jbuilder

```diff
@@ -4,14 +4,16 @@ json.version cookbook_version.version
 json.average_rating nil
 json.cookbook api_v1_cookbook_url(cookbook)
 json.file api_v1_cookbook_version_download_url(cookbook_version.cookbook, cookbook_version)
-json.quality do
-  json.foodcritic do
-    json.failed @cookbook.foodcritic_failure
-    json.feedback @cookbook.foodcritic_feedback
-  end
-  json.collaborator do
-    json.failed @cookbook.collaborator_failure
-    json.feedback @cookbook.collaborator_feedback
+if !@cookbook_version_metrics.nil? && @cookbook_version_metrics.any?
```

Comment:
We can consider using `.present?` in a future refactor (if we don't opt to iterate).